packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -23,7 +23,9 @@
   "categories": ["Languages"],
   "dependencies": {
     "lwc-language-server": "0.14.1",
-    "vscode-languageclient": "3.5.0"
+    "vscode-languageclient": "3.5.0",
+    "eslint": "4.15.0",
+    "eslint-plugin-lwc": "0.2.0"
```

Comment:
These are the versions of eslint and estlint-plugin-lwc that we currently bundle.
Unzipped on disk, they total about 34 MB.
In the .vsix that is produced, everything can be compressed to 9 MB so it's less of an issue.